modules/express/src/typedRoutes/schemas/error.ts

```
@@ -0,0 +1,8 @@
+import * as t from 'io-ts';
+
+export const BitgoExpressError = t.type({
+  message: t.string,
+  name: t.string,
+  bitgoJsVersion: t.string,
+  bitgoExpressVersion: t.string,
```

Are these values always present on the response? should some of these be optional?

---

We have express error type as this defined in clientRoutes catch all handler. 
```
  result = _.extend({}, result, {
    message: err.message,
    name: err.name || 'BitGoExpressError',
    bitgoJsVersion: version,
    bitgoExpressVersion: pjson.version,
  });

```